packages/mdctl-axon-tools/lib/StudyManifestTools.js

```diff
@@ -125,6 +125,12 @@ class StudyManifestTools {
 
   async getStudyManifest() {
```

Comment:
Also noticed that we have some code duplication for the other mdctl tasks, check
- getTasksManifest
- getConsentsManifest

For both cases  we could split the logic too, this will be really convenient for the next release that we are going to be using those other more granular exports

Reply:
Hi @nahueld is it ok for you if I create another Jira to keep track of this as I guess we probably don't want to overlap with the original requirement?

Reply:
ok let's update the ticket to reflect this